docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Tetragon Metrics"
+weight: 1
+icon: "overview"
+description: "Fetching and understanding Tetragon metrics"
+---
+
+# Kubernetes
+
+Tetragon, when installed with Helm as mentioned in 
+[Deploying on Kubernetes]({{< ref "/docs/getting-started/deployment/kubernetes" >}}) creates a pod with the `tetragon-` prefix. It also creates a service named `tetragon` that exposes metrics on the specified port.
+
+## Getting metrics port
+
+Check if the `tetragon` service exists:
+
+```shell-session
+kubectl get services tetragon -n kube-system
+```
+
+The output should be similar to:
+```
+NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+tetragon   ClusterIP   10.96.54.218   <none>        2112/TCP    3m
+```
+
+{{< note >}}
+In the previous output it shows, 2112 is the port on which the service is 
+listening. It is also the port on which Tetragon listens for its metrics 
+server with the default Helm values.
+{{< /note >}}
+
+## Port Forwarding
+
+To forward the metrics port locally, use `kubectl port forward`:
+
+```shell-session
+kubectl -n kube-system port-forward service/tetragon 2112:2112
+```
+
+# Package
+
+Tetragon, when installed via release packages as mentioned in [Package Deployment]({{< ref "/docs/getting-started/deployment/package" >}}). It can be interacted with `tetragon` command. By default, metrics are disabled, which can be enabled using `--metrics-server` flag, by specifying the address.
+
+Alternatively, the metrics address can also be set by creating a file under
+`etc/tetragon/tetragon.conf.d/metrics-server`. 
+
```

Comment:
here you are missing the leading `/`  it is `/etc/...` and can you re-word please: by creating the `/etc/tetragon/tetragon.conf.d/metrics-server` drop-in configuration with the address where to listen as content.

Like it is state in the original documentation or examples ;-)